src/strokes.ts

```diff
@@ -130,114 +130,87 @@ export class Stroke extends Modifier {
-        arrowShiftX = -3;
-        textShiftX = this.xShift + arrowShiftX - 2;
-        if (isStaveNote(note)) {
-          topY += 1.5 * lineSpace;
-          if ((botY - topY) % 2 !== 0) {
-            botY += 0.5 * lineSpace;
-          } else {
-            botY += lineSpace;
-          }
-          arrowY = topY - lineSpace;
-          textY = botY + lineSpace + 2;
-        } else {
-          topY += 1.5 * lineSpace;
-          botY += lineSpace;
-          arrowY = topY - 0.75 * lineSpace;
-          textY = botY + 0.25 * lineSpace;
-        }
+        arrow = '\ueb78' /*arrowheadBlackUp*/;
+        arrowY = topY;
+        topY -= lineSpace / 2;
+        botY += lineSpace / 2;
         break;
+      case Stroke.Type.BRUSH_UP:
       case Stroke.Type.ROLL_UP:
       case Stroke.Type.RASQUEDO_UP:
-        arrow = 'arrowheadBlackDown';
-        arrowShiftX = -4;
-        textShiftX = this.xShift + arrowShiftX - 1;
-        if (isStaveNote(note)) {
-          arrowY = lineSpace / 2;
-          topY += 0.5 * lineSpace;
-          if ((botY - topY) % 2 === 0) {
-            botY += lineSpace / 2;
-          }
-          arrowY = botY + 0.5 * lineSpace;
-          textY = topY - 1.25 * lineSpace;
-        } else {
-          topY += 0.25 * lineSpace;
-          botY += 0.5 * lineSpace;
-          arrowY = botY + 0.25 * lineSpace;
-          textY = topY - lineSpace;
-        }
+        arrow = '\ueb7c' /*arrowheadBlackDown*/;
+        arrowY = botY + lineSpace;
+        topY -= lineSpace / 2;
         break;
       case Stroke.Type.ARPEGGIO_DIRECTIONLESS:
-        topY += 0.5 * lineSpace;
-        botY += lineSpace; // * 0.5 can lead to slight underlap instead of overlap sometimes
+        topY -= lineSpace / 2;
+        botY += lineSpace / 2;
         break;
       default:
         throw new RuntimeError('InvalidType', `The stroke type ${this.type} does not exist`);
     }
 
-    // The first letter is capitalized.
-    // We later prepend 'stroke' to create a camelCased glyph category: 'strokeStraight' or 'strokeWiggly'.
-    // See: common_metrics.ts
-    let type = 'Straight';
-
     // Draw the stroke
     if (this.type === Stroke.Type.BRUSH_DOWN || this.type === Stroke.Type.BRUSH_UP) {
       ctx.fillRect(x + this.xShift, topY, 1, botY - topY);
     } else {
-      type = 'Wiggly';
-      if (isStaveNote(note)) {
-        for (let i = topY; i <= botY; i += lineSpace) {
-          Glyph.renderGlyph(ctx, x + this.xShift - 4, i, this.renderOptions.fontScale, 'vexWiggleArpeggioUp');
-        }
+      const lineGlyph =
+        arrow === '\ueb7c' /*arrowheadBlackDown*/ ? '\ueaa9' /*wiggleArpeggiatoUp*/ : '\ueaaa'; /*wiggleArpeggiatoDown*/
+      let txt = '';
+      const el = new Element();
+      // 2 pixels overlap of arpeggiato glyphs
+      for (; el.getWidth() < botY - topY; ) {
+        txt += lineGlyph;
+        el.setText(txt);
+        el.measureText();
+      }
+      if (
+        this.type === Stroke.Type.RASQUEDO_DOWN ||
```

Comment:
This has probably been in the code base for a while, but I think Rasgueado is misspelled. :-)

Our typo doesn't match any of the alternative spelling on the wikipedia page: https://en.wikipedia.org/wiki/Rasgueado

The closest one is Rasqueado but we are still missing a letter a. I would suggest a separate PR to fix this (to use the official wikipedia spelling).

Reply:
It has been around since 3.0.9, so we could (maybe) consider supporting the typo in addition to the more correct spelling. I'm actually OK with just breaking backwards compatibility, since lots of breaking changes are happening anyways. We just need to document it.

https://github.com/0xfe/vexflow/blob/00ec15c67ff333ea49f4d3defbd9e22374c03684/src/strokes.js#L21-L22

Reply:
And what option is the prefered one? Rasgueado?

Reply:
OK  already in an issue, no action then,

Reply:
Yeah we might as well follow the wikipedia spelling :-)